### ISSUE TITLE
Is Pyrosm dead?

### ISSUE THREAD

**Issue by awa5114** [HTenkanen/pyrosm#240]
Is this library still being maintained? I see there haven't been any new comments since late 2023 and the last closed issue is also from 2023. I am relatively new to processing osm data with python and would liketo know whether I can still rely on this library or whether I should turn to something a bit lower level like[ `pyosmium`](https://github.com/osmcode/pyosmium) which is much less user friendly but has regular updates?

**Comment by HTenkanen** [comment]
Hi! I would not call us dead yet, although unfortunately I have had very limited time to maintain this library. However, the key functionalities of `pyrosm` should work regardless of the past inactivity in the repo. Depending on what you need, in addition to `pyosmium` you could also check the `osmnx` library which provides easy-to-use interface to fetch OSM data via Overpass API.

**Comment by awa5114** [comment]
Hi @HTenkanen  thanks for your response. As far as I understand it, `osnmx` is more suited to network data (streets, waterways) and is probably slower than `pyrosm` seeing as it uses Overpass. Does `pyrosm` use `osmium`  under the hood? If not how is it able to communicate with PBF files? What can pyrosm do which osmium can not and vice versa?

Thanks in advance.

**Comment by gboeing** [comment]
@amine-aboufirass yes OSMnx is slower than pyrosm as it retrieves data from either a) Overpass or b) local .osm XML files (with option b being much faster than option a, but still slower than reading from a PBF file). However, OSMnx is capable of retrieving any OSM features as a GeoPandas GeoDataFrame and is not limited to network data/analysis.

**Comment by simonprovost** [comment]
@HTenkanen Maybe push a call for part time maintainers? maybe some of the folks using PyrOSM might want to jump into the adventure? Just an idea because in anyway while OSMNX from @gboeing is amainzg, PyrOSM should not get too depreceated as it has a lot of value too.

**Comment by HTenkanen** [comment]
Hi all! Thanks for your worries, comments and support! ❤️ I'm happy to close this issue with a serious push in the past month to add new functionalities and fixing numerous bugs reported by the users. I.e. `pyrosm` is again up and running and the plan is to gather a community around this, so that similar inactivity period would not happen ever again. Check the latest pyrosm v0.11.0 out and read the documentation here: 
https://pyrosm.readthedocs.io
